src/otx/core/model/detection.py

```diff
@@ -461,11 +467,14 @@ def _forward_explain_detection(
 
     def get_explain_fn(self) -> Callable:
         """Returns explain function."""
-        from otx.algo.detection.heads import SSDHead
+        from otx.algo.detection.heads.ssd_head import SSDHeadModule
         from otx.algo.explain.explain_algo import DetClassProbabilityMap
 
         # SSD-like heads also have background class
-        background_class = hasattr(self.model, "bbox_head") and isinstance(self.model.bbox_head, SSDHead)
+        background_class = hasattr(self.model, "bbox_head") and isinstance(
+            self.model.bbox_head,
+            SSDHeadModule,
+        )  # TODO (sungchul): revert module's name?
```

Comment:
Are we sure `SSDHead` is the only one that fits the bill?

Reply:
I have no idea about xai, @GalyaZalesskaya could you give us some advice?

Reply:
Background class is needed and used only to explain SSD model among all other detection models